orc8r/cloud/go/obsidian/access/tests/rest_middleware_test.go

```diff
@@ -83,18 +93,18 @@ func TestAuthMiddleware(t *testing.T) {
 		{"GET", fmt.Sprintf("%s%s", urlPrefix, RegisterNetworkV1), test_utils.TestRootUsername, rootToken, http.StatusOK},
 		{"GET", fmt.Sprintf("%s%s", urlPrefix, "/malformed/url"), test_utils.TestRootUsername, rootToken, http.StatusOK},
 		{"PUT", fmt.Sprintf("%s%s", urlPrefix, "/malformed/url"), test_utils.TestRootUsername, rootToken, http.StatusOK},
-		{"GET", fmt.Sprintf("%s%s", urlPrefix, tenantsh.TenantInfoURL), test_utils.TestRootUsername, rootToken, http.StatusOK},
-		{"POST", fmt.Sprintf("%s%s", urlPrefix, tenantsh.TenantInfoURL), test_utils.TestRootUsername, rootToken, http.StatusOK},
+		{"GET", fmt.Sprintf("%s%s%s%s", urlPrefix, TenantRootPathV1, obsidian.UrlSep, strconv.Itoa(test_utils.TestTenantId)), test_utils.TestRootUsername, rootToken, http.StatusOK},
+		{"POST", fmt.Sprintf("%s%s%s%s", urlPrefix, TenantRootPathV1, obsidian.UrlSep, strconv.Itoa(test_utils.TestTenantId)), test_utils.TestRootUsername, rootToken, http.StatusOK},
```

Comment:
Only admins should be able to make edits to the tenants path

The "tenant" policy type is just a dynamic alias for a list of network IDs

Reply:
Ok so should I just put in a const like adminOnlyPaths?

Reply:
Uh nah it's okay if someone creates e.g. a path-specific policy giving someone access to the tenants endpoints. But it's more in reference to my other comment about the purpose of the tenant-scoped policies. Hopefully my msg with the proposed proto changes will help align us